src/components/PokemonItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { View, Text, StyleSheet } from "react-native";
+
+type PokemonItemProps = {
+  url?: string; // Will be used later
```

Comment:
If it is not used now, why add it?

Reply:
true, will not leave things that might be used in the future